data/human/human missions.txt

```diff
@@ -5894,7 +5894,7 @@ mission "Quicksilver Mixup 2"
 	on visit
 		dialog `You have reached <planet>, but your escort, loaded with the shipment, has not arrived. Better depart and wait for your escorts to arrive in this star system.`
 	on complete
-		dialog `You contact the spaceport authorities and they direct you to the respective landing pad, where the post office staff is already waiting. They transfer the cargo and you prepare to head back to <origin>. Again, you have 10 days to make the delivery.`
+		dialog `You contact the spaceport authorities and they direct you to the landing pad, where the post office staff is already waiting. They transfer the cargo and you prepare to head back to <origin>. Again, you have 10 days to make the delivery.`
```

Comment:
This is unnecessary; the only purpose it serves is having one less word in the sentence. It's not a fix.

Reply:
I had had it as "right landing pad", but @tibetiroka didn't like the work right.

Reply:
So which do I do? "right landing pad" or "respective landing pad"? Or just "landing pad"?

Reply:
I think "respective" is confusing here, so I'd go for "right landing pad" or "a landing pad".